recipes.js

```diff
@@ -2,9 +2,67 @@ const mongoose = require('mongoose');
 const Schema   = mongoose.Schema;
 const data = require('./data.js');
 
+const Recipe = mongoose.Schema 
```

Comment:
The Recipe variable will make the connection between your recipeShema and the Recipes collection in your database, you can declare it like that : 
```js
const Recipe = mongoose.model("Recipe", recipeSchema)
```

Reply:
Also you can declare it after you have declared your recepeSchema